### DISCUSSION TITLE
input type=text

### DISCUSSION THREAD

**Discussion by Manubi** [glacambre/firenvim#1568] [Ideas]
Hi everyone.
I didn't find an answer why `<input type="text"/>` is not supported? Just curious to know why.

**Comment by alerque** [comment]
What says it doesn't? Did you try appending `, input[type="text"]` to your selector setting? You might find that is too aggressive and only enable that selector on specific sites or even specific input fields by ID, but it does work.

**Comment by glacambre** [comment]
Yep, as alerque explained, you can configure the elements Firenvim should appear on by configuring Firenvim's [selector](https://github.com/glacambre/firenvim#configuring-what-elements-firenvim-should-appear-on) option. A fun fact: very early Firenvim versions used to take over input elements by default! It was disabled because it [turned out to be far more annoying than expected](https://github.com/glacambre/firenvim/issues/17). I still want to someday implement a "[seamless mode](https://github.com/glacambre/firenvim/issues/1525)" closer to what [pterosaur](https://github.com/ardagnir/pterosaur) did, meaning that Firenvim would not open a Neovim frame, you would instead get your regular text input element but all keybindings would work as they do in Neovim's insert mode. I think this would be perfect for input elements. But I haven't started work on that yet :)

**Reply by alerque** [reply]
That sounds amazing, and perfect one one-line editing where the UI really isn't serving much of a purpose.

**Reply by Manubi** [reply]
this sound's cool. hope to see it soon as the :wq get's a little annoying for short searches.

**Reply by alerque** [reply]
You know you can bind anything you like to that right? You could even bind `<Esc><Esc>` to save and drop out of Firenvim or (as I used to do) just enable autosave and don't even bother closing it.

**Reply by Manubi** [reply]
can you quickly tell me how would I do that? thanks!
